client/src/shims-tsx.d.ts

```diff
@@ -0,0 +1,13 @@
+import Vue, { VNode } from 'vue';
+
+declare global {
+  namespace JSX {
+    // tslint:disable no-empty-interface
+    interface Element extends VNode {}
+    // tslint:disable no-empty-interface
+    interface ElementClass extends Vue {}
+    interface IntrinsicElements {
+      [elem: string]: any
+    }
+  }
+}
```

Comment:
Hey @AlmightyYakob should we just remove this whole file?  We aren't using JSX and I didn't notice before but that's all these shims are here for.

Reply:
Since we're not using JSX currently I'm okay with removing it, this is mainly boilerplate stuff that we could come up with again anyway right?